src/autoscaler/api/policyvalidator/policy_validator_test.go

```diff
@@ -2343,7 +2343,7 @@ var _ = Describe("PolicyValidator", func() {
 					"threshold":90,
 					"operator":">=",
 					"cool_down_secs":300,
-					"adjustment":"+1"
+					"adjustment":"+10%"
```

Comment:
please add a new test case to validate "%"  as an adjust parameter , instead of modifying current "good" path.  We may different good path here.